src/components/Scheme/SchemeContainer.tsx

```diff
@@ -53,6 +55,28 @@ const mapDispatchToProps = {
 
 class SchemeContainer extends React.Component<IProps, null> {
 
+  private getSchemeIsActive(scheme: ISchemeState): boolean {
+    if (scheme.name === "SchemeRegistrar") {
+      /**
+       * If SchemeRegistrar has only one or the other active, then we'll deal with that case in
+       * `CreateSchemeRegistrarProposal`
+       */
+      return moment((scheme as any).schemeRegistrarParams.voteRegisterParams.activationTime).isSameOrBefore(moment()) ||
+             moment((scheme as any).schemeRegistrarParams.voteRemoveParams.activationTime).isSameOrBefore(moment());
+    } else {
+      let name = `${scheme.name[0].toLowerCase()}${scheme.name.slice(1)}`;
+      if (name === "genericScheme") {
+        if (scheme.uGenericSchemeParams) {
+          name = "uGenericScheme";
+        }
+      }
+      /**
+     * assumes the voting machine is GP and its params can be found at scheme.<schemeName>Params.voteParams
+     */
+      return moment((scheme as any)[`${name}Params`].voteParams.activationTime).isSameOrBefore(moment());
```

Comment:
As you note in the PR description, this will fail if a scheme does not have these parameters set. It would be better (more future-proof, in any case) to handle that case explicitly - we can in that case just assume that the scheme is active....

Reply:
Done